content/en/docs/concepts/extend-kubernetes/poseidon-firmament-alternate-scheduler.md

```diff
@@ -23,9 +23,7 @@ Firmament models the scheduling problem as a constraint-based optimization over
 
 Poseidon-Firmament scheduler runs alongside the default Kubernetes scheduler as an alternate scheduler. You can run multiple schedulers run simultaneously.
 
-## Key Advantages
-
-### Flow graph scheduling based Poseidon-Firmament scheduler provides the following key advantages:
+Flow graph scheduling based Poseidon-Firmament scheduler provides the following key advantages:
```

Comment:
Hi @sftim . I read the page. Nearly ready. Take a look.
This change needs a new line at line 27 to render the list 😸 
Suggest rewording line 24 to something like:You can simultaneously run multiple schedulers.